.github/ISSUE_TEMPLATE/docs-task.yml

```
@@ -0,0 +1,44 @@
+name: "📝 Documentation"
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)